draft-birkholz-scitt-software-use-cases.md

```diff
@@ -167,26 +167,34 @@ There is no standardized way to:
 * to check for the trustworthiness of a provider
 * to check the integrity of modifications or transformations done by a provider
 
-## Trustworthy Reference Values for After-The-Fact Analysis
+## Verifying Firmware Provenance for large and complex systems
```

Comment:
Verifying Firmware Provenance for a large and complex systems

Reply:
Let's check with Monty on the title on Monday. I'll check for his availability.

Reply:
Post Boot Firmware Provenance